src/coreclr/inc/bitposition.h

```diff
@@ -12,47 +12,35 @@
 //
 // Notes:
 //    'value' must have exactly one bit set.
-//    The algorithm is as follows:
-//    - PRIME is a prime bigger than sizeof(unsigned int), which is not of the
-//      form 2^n-1.
-//    - Taking the modulo of 'value' with this will produce a unique hash for all
-//      powers of 2 (which is what "value" is).
-//    - Entries in hashTable[] which are -1 should never be used. There
-//      should be PRIME-8*sizeof(value) entries which are -1 .
 //
 inline
 unsigned            BitPosition(unsigned value)
```

Comment:
This LGTM, but its worth calling out that this is "effectively" doing `TrailingZeroCount`, but without handling `0`.

It might be worth centralizing these "primitive bit operation" methods somewhere and ensuring they are generally reusable.

Reply:
Opened tracking issue https://github.com/dotnet/runtime/issues/69401